db/models/ally.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Ally = db.define('allies', {
+  username: Sequelize.STRING,
+  phone_number: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  email: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      isEmail: true
+    }
+  },
+  job_title: Sequelize.STRING,
+  company: Sequelize.STRING,
+  industry: Sequelize.ARRAY(Sequelize.STRING),
+  max_clients: Sequelize.INTEGER
+}, {
+  indexes: [
+    { unique: false, fields: ['industry'] }
```

Comment:
I thought it was just weird that this was one of the few indexes I saw...I'd think we'd index email and/or sign-in info before this. Something else to chat about potentially?